README.md

```diff
@@ -1,5 +1,8 @@
 # semantic-release-github-pullrequest
-[**semantic-release**](https://github.com/semantic-release/semantic-release) plugin to create a GitHub pullrequest after a release
+
+This plugin is a plugin for [**semantic-release**](https://github.com/semantic-release/semantic-release).
+
+It automatically creates a pull request containing the release notes of a newly published release in your repository, as long as you follow the Semantic release convention.
```

Comment:
It's more generic than `creates a pull request containing the release notes of a newly published release`, I'll propose:
```suggestion
It automatically creates a pull request containing changes for any files you want to publish in your repository, like release notes of your newly published release.
```